### DISCUSSION TITLE
Warning: Extra attributes from the server: wtx-context

### DISCUSSION THREAD

**Discussion by Banchereau** [vercel/next.js#39513] [Help]
Hello
I have this warning in my console:
next-dev.js?3515:20 Warning: Extra attributes from the server: wtx-context.
All I know is that warning is caused by my search component. Because if I remove this component, the warning disappears. Here is the code:
![search](https://user-images.githubusercontent.com/80891922/184197081-4a1b195d-3041-4dc6-b817-cd50526640c4.jpg)

**Comment by gaston-flores** [comment]
Maybe a browser extension or AV is adding this attribute to `input`? Have you tried visiting the page in incognito or from another computer?

**Reply by Banchereau** [reply]
Well from another computer, it seems a bit difficult as it's a developpement website.
I've tried on mozilla and I didn't see any warning.
So, you are right. This is related to the browser.
Thank's a lot.

**Reply by mremolt** [reply]
That attribute is added by Bitdefender, when you deactivate their extension (temporarily), it does not show up.
